manager/testdata/blueprint.yaml

```diff
@@ -27,7 +27,8 @@ spec:
                 endpoint: s3.eu-gb.cloud-object-storage.appdomain.cloud
                 object_key: small.kp.encrypted.parq
             type: 2
-            credentialLocation: /v1/m4d/dataset-creds/m4d-test-bucket
+            vault:
+              secretPath: /v1/m4d/dataset-creds/m4d-test-bucket
```

Comment:
vault fields also includes the following sub-feilds:
   vault:
      address: http://vault.m4d-system:8200
      authPath: /v1/auth/kubernetes/login
      role: module
      secretPath: "/v1/kubernetes-secrets/secret-name?namespace=default"